dashboard/static/css/home.css

```diff
@@ -0,0 +1,3 @@
+form#add-push-application input[type="submit"] {
+    display: block;
+}
```

Comment:
How would you feel about leaving all styles in a single stylesheet for the whole site?